cmd/run.go

```diff
@@ -31,8 +31,8 @@ func RunCmd(lg log.Logger, mt *metrics.StatsdMonitor) *cobra.Command {
 			}
 
 			runs := runner.RunMultiple(recipes)
-
 			for _, run := range runs {
+				lg.Debug("recipe details", "config", run.Recipe)
```

Comment:
@StewartJingga any reason we are using the debug now while earlier we weren't. Just wanted to know if odpf/salt has brought any change that is being addressed here.

Reply:
odpf/salt has nothing to do with it, the reason is because I need information about the recipes that are being run but only when log level is debug.

Reply:
oh okay, got it